wsutil/reader_test.go

```diff
@@ -168,6 +168,32 @@ func TestReaderNextFrameAndReadEOF(t *testing.T) {
 
 }
 
+func TestMaxFrameSize(t *testing.T) {
+	var buf bytes.Buffer
+	f := ws.NewTextFrame([]byte("small frame")) // 11 bytes
```

Comment:
what about to extract 11 or 10 to the `const` or just a `var` ?

Reply:
I have a better idea ;)

Reply:
Yep, exactly what the next commits are :)